cypress/fixtures/api/insights-results-aggregator/v2/clusters.json

```diff
@@ -5,205 +5,378 @@
       "total_hit_count": 9,
-      "hits_by_total_risk": { "1": 3, "2": 0, "3": 5, "4": 1 }
+      "hits_by_total_risk": {
+        "1": 3,
+        "2": 0,
+        "3": 5,
+        "4": 1
+      },
+      "cluster_version": "4.4.24"
     },
     {
       "cluster_id": "849f9d51-f22a-444d-a601-d4512307b7da",
       "cluster_name": "s8yl8o7kr 5cb21ou71",
       "last_checked_at": "2019-07-19T08:18:12.495Z",
       "total_hit_count": 23,
-      "hits_by_total_risk": { "1": 5, "2": 13, "3": 2, "4": 3 }
+      "hits_by_total_risk": {
+        "1": 5,
+        "2": 13,
+        "3": 2,
+        "4": 3
+      },
+      "cluster_version": "4.17.9"
     },
     {
       "cluster_id": "fc603601-0ff8-45e4-b0f3-c7f76d2ef36b",
       "cluster_name": "gsbq8pthf xah3olxhz",
       "last_checked_at": "2019-02-09T13:43:12.855Z",
       "total_hit_count": 0,
-      "hits_by_total_risk": { "1": 0, "2": 0, "3": 0, "4": 0 }
+      "hits_by_total_risk": {
+        "1": 0,
+        "2": 0,
+        "3": 0,
+        "4": 0
+      },
+      "cluster_version": "4.5.5"
     },
     {
       "cluster_id": "deba971e-8f8c-4e37-8e0a-4655523250de",
       "cluster_name": "",
       "last_checked_at": "2021-02-23T11:13:10.518Z",
       "total_hit_count": 20,
-      "hits_by_total_risk": { "1": 0, "2": 9, "3": 9, "4": 2 }
+      "hits_by_total_risk": {
+        "1": 0,
+        "2": 9,
+        "3": 9,
+        "4": 2
+      },
+      "cluster_version": "4.7.7"
     },
     {
       "cluster_id": "707e82e4-34f3-4ad8-8814-28cadd718e1a",
       "cluster_name": "foo baz",
       "last_checked_at": "2020-05-15T23:10:49.825Z",
       "total_hit_count": 41,
-      "hits_by_total_risk": { "1": 14, "2": 18, "3": 7, "4": 2 }
+      "hits_by_total_risk": {
+        "1": 14,
+        "2": 18,
+        "3": 7,
+        "4": 2
+      },
+      "cluster_version": "4.1.2"
     },
     {
       "cluster_id": "d5b3b06a-31af-4ebe-8192-2fde8a811a62",
       "cluster_name": "",
       "last_checked_at": "2021-02-21T05:12:59.441Z",
       "total_hit_count": 0,
-      "hits_by_total_risk": { "1": 0, "2": 0, "3": 0, "4": 0 }
+      "hits_by_total_risk": {
+        "1": 0,
+        "2": 0,
+        "3": 0,
+        "4": 0
+      },
+      "cluster_version": ""
     },
     {
       "cluster_id": "cc59cabb-cb40-4a7e-8665-feb822a210e3",
       "cluster_name": "",
       "last_checked_at": "2019-12-18T14:21:12.020Z",
       "total_hit_count": 19,
-      "hits_by_total_risk": { "1": 11, "2": 8, "3": 0, "4": 0 }
+      "hits_by_total_risk": {
+        "1": 11,
+        "2": 8,
+        "3": 0,
+        "4": 0
+      },
+      "cluster_version": "4.17.9"
     },
     {
       "cluster_id": "0abb45ac-3ade-49c7-a7d4-ec2af0ee21db",
       "cluster_name": "y7gz4faz1 k6rqb02yo",
       "total_hit_count": 0,
-      "hits_by_total_risk": { "1": 0, "2": 0, "3": 0, "4": 0 }
+      "hits_by_total_risk": {
+        "1": 0,
+        "2": 0,
+        "3": 0,
+        "4": 0
+      },
+      "cluster_version": "4.3.24"
```

Comment:
Isn't this provided as meta.cluster_version?

Reply:
No, here, you can check it: https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/server/api/v2/openapi.json#L1444.